packages/core-app-api/src/apis/implementations/auth/saml/types.ts

```diff
@@ -23,13 +23,11 @@ import { z } from 'zod';
  * @public
  */
 export type SamlSession = {
-  userId: string;
   profile: ProfileInfo;
   backstageIdentity: BackstageIdentity;
```

Comment:
Keeping this required as it won't make sense to have a SAML provider without a sign-in resolver